tests/error/misc_errors/state_result_missing_tag.err

```diff
@@ -0,0 +1,8 @@
+Error: Type mismatch (at $FILE:12:17)
+   | 
+10 | def main() -> None:
+11 |     q1 = qubit()
+12 |     state_result(q1)
+   |                  ^^ Expected expression of type `str`, got `qubit`
```

Comment:
(While it would be nice to be more specific about the missing tag this is the same as result at the moment)